README.md

```diff
@@ -356,14 +356,14 @@ Configuration Add_FirewallRuleToExistingGroup
         {
             Name                  = "MyFirewallRule"
             DisplayName           = "My Firewall Rule"
-            DisplayGroup          = "My Firewall Rule Group"
+            Group          = "My Firewall Rule Group"
```

Comment:
nit: can you get the spacing to line up on this line? Always drives me nuts when it doesn't. :grin: